certbot/tests/util_test.py

```diff
@@ -45,34 +45,19 @@ def test_failure(self, mock_popen):
         self.assertRaises(errors.SubprocessError, self._call, ["test"])
 
 
-class ExeExistsTest(unittest.TestCase):
-    """Tests for certbot.util.exe_exists."""
-
-    @classmethod
-    def _call(cls, exe):
-        from certbot.util import exe_exists
-        return exe_exists(exe)
-
-    @mock.patch("certbot.util.os.path.isfile")
-    @mock.patch("certbot.util.os.access")
-    def test_full_path(self, mock_access, mock_isfile):
-        mock_access.return_value = True
-        mock_isfile.return_value = True
-        self.assertTrue(self._call("/path/to/exe"))
-
-    @mock.patch("certbot.util.os.path.isfile")
-    @mock.patch("certbot.util.os.access")
-    def test_on_path(self, mock_access, mock_isfile):
-        mock_access.return_value = True
-        mock_isfile.return_value = True
-        self.assertTrue(self._call("exe"))
-
-    @mock.patch("certbot.util.os.path.isfile")
-    @mock.patch("certbot.util.os.access")
-    def test_not_found(self, mock_access, mock_isfile):
-        mock_access.return_value = False
-        mock_isfile.return_value = True
-        self.assertFalse(self._call("exe"))
+class WhichTest(unittest.TestCase):
+    @mock.patch('certbot.util._is_exe')
```

Comment:
nit: It's nice to be consistent for strings and stick to either single or double quotes. This is specified by both of our [style](https://www.python.org/dev/peps/pep-0008/#string-quotes) [guides](https://google.github.io/styleguide/pyguide.html?showone=Strings#Strings).